metadata/metadata.go

```diff
@@ -168,9 +168,8 @@ type ForwardMetadata struct {
 	// Pipeline of operations that may be applied to the metric.
 	Pipeline applied.Pipeline
 
-	// Metric source id. This usually refers to the source metric id producing
-	// this forward metric.
-	SourceID []byte
+	// Metric source id that refers to the unique id of the source producing this metric.
```

Comment:
Mention it's the shard of the raw ID?

Reply:
I'm intentionally keeping it vague. It is currently the shard, but *may* be something else. There's no mentioning of shard in this package so it would be strange to bring it up.